### ISSUE TITLE
Add all current gnu args to wiki

### ISSUE THREAD

**Issue by ew3361zh** [RuminantFarmSystems/RuFaS#934]
Add a section to the wiki which overviews the available gnu args and what each of their effects is.

**Comment by ew3361zh** [comment]
This is currently implemented here: https://github.com/RuminantFarmSystems/MASM/wiki/RuFaS-Command-Line-Arguments

But I think we should wait to mark this as complete until PR #878 is merged into main. There are 3 new gnu args included in that PR which should also be added to the wiki when they are functional.

**Comment by ew3361zh** [comment]
Added new args from PR #958 but still waiting on #878.

**Comment by Isgotkowitz** [comment]
@allisterakun will update GNU args in #878 before it gets closed.

**Comment by ew3361zh** [comment]
#878 has GNU args added, closes this issue.
